src/VecSim/algorithms/hnsw/hnsw.h

```diff
@@ -580,11 +575,11 @@ void HNSWIndex<DataType, DistType>::emplaceToHeap(
 template <typename DataType, typename DistType>
 template <bool has_marked_deleted, typename Identifier>
 void HNSWIndex<DataType, DistType>::processCandidate(
-    idType curNodeId, ElementGraphData *cur_meta, const void *query_data, size_t layer, size_t ef,
-    tag_t *elements_tags, tag_t visited_tag,
-    vecsim_stl::abstract_priority_queue<DistType, Identifier> &top_candidates,
+    idType curNodeId, const void *query_data, size_t layer, size_t ef, tag_t *elements_tags,
+    tag_t visited_tag, vecsim_stl::abstract_priority_queue<DistType, Identifier> &top_candidates,
     candidatesMaxHeap<DistType> &candidate_set, DistType &lowerBound) const {
 
+    ElementGraphData *cur_meta = getGraphDataByInternalId(curNodeId);
```

Comment:
meta? :)

Reply:
:)
This PR is prior to the renaming one